cmd/helm/search_hub.go

```diff
@@ -70,6 +71,8 @@ func newSearchHubCmd(out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.StringVar(&o.searchEndpoint, "endpoint", "https://hub.helm.sh", "Hub instance to query for charts")
 	f.UintVar(&o.maxColWidth, "max-col-width", 50, "maximum column width for output table")
+	f.BoolVar(&o.listRepoURL, "list-repo", false, "print charts repository URL")
```

Comment:
why not use `list-repo-url`?

Reply:
Thank you for your suggestion. I changed the flag, please take a look.